skimage/filters/thresholding.py

```diff
@@ -212,7 +214,18 @@ def threshold_adaptive(image, block_size, method='gaussian', offset=0,
     elif method == 'median':
         ndi.median_filter(image, block_size, output=thresh_image, mode=mode)
 
-    return image > (thresh_image - offset)
+    return thresh_image - offset
+
+
+
+@deprecated('threshold_local')
+def threshold_adaptive(image, block_size, method='gaussian', offset=0,
+                       mode='reflect', param=None):
+    warn('The return value of `threshold_local` is not the same as that '
+         'of `threshold_adaptive`.')
```

Comment:
Perhaps we can be more precise here by saying that threshold_adaptive returns a thresholded image while threshold_local return the threshold values.